functions/src/api/v1/group/subscribeToGroup.ts

```diff
@@ -0,0 +1,67 @@
+import { wrap } from 'async-middleware'
+import { Request, Response } from 'express'
+import { UID } from '../../../types/uid'
+import { GroupDao } from '../../../db/GroupDao'
+import { GroupId } from '../../../types/Group'
+import { assertNewResourceCreationGranted } from '../subscription/assertNewResourceCreationGranted'
+import { messaging } from '../../../firebase/firebase'
+import { MessagingServerError } from '../../errors/HttpError'
+
+/**
+ * @swagger
+ * /v1/groups/{groupId}/subscribe:
+ *   post:
+ *     description: Subscribe a device from a user to a group in order to receive push notifications.<br/>Warning! The user does not need to be in the group yet (it will be automatically added), but this will change later on. Also, the group owner accounts/subscription status need to be valid.
```

Comment:
I would have said: `Subscribe a user's device to a group`, or `Subscribe a user to a group`